src/frontend/src/optimizer/plan_node/logical_expand.rs

```diff
@@ -76,6 +80,7 @@ impl LogicalExpand {
 
     fn derive_schema(input_schema: &Schema) -> Schema {
         let mut fields = input_schema.clone().into_fields();
+        fields.extend(fields.clone());
```

Comment:
Please add some docs for `struct LogicalExpand` to explain the new definition.

Reply:
okk